plenum/persistence/req_id_to_txn.py

```diff
@@ -0,0 +1,50 @@
+from hashlib import sha256
+from typing import Optional
+
+from plenum.persistence.kv_store import KVStoreLeveldb
+
+
+class ReqIdrToTxn:
+    """
+    Stores a map from client identifier, request id tuple to transaction
+    sequence number
+    """
+    def __init__(self, *args, **kwargs):
+        raise NotImplementedError
+
+    def add(self, identifier, reqId, seqNo):
+        raise NotImplementedError
+
+    def get(self, identifier, reqId):
+        raise NotImplementedError
+
+
+class ReqIdrToTxnLevelDB(ReqIdrToTxn):
```

Comment:
I usually follow this, if the interface and implementation are small and are going to be small keep them in the same file but if we know the interface and implementation will grow big, then move to a separate file, tell me if you disagree